release/rllib_tests/learning_tests/yaml_files/appo/appo-pongnoframeskip-v4.yaml

```diff
@@ -7,6 +7,8 @@ appo-pongnoframeskip-v4:
         timesteps_total: 5000000
     stop:
         time_total_s: 1800
+    # TODO (Kourosh): Torch and tf2 do not learn as good as tf. Why?
+    frameworks: ["tf"]
```

Comment:
The reason why torch does not learn this task for this test is simply speed (very similar issue with the corresponding PPO test). After a deeper investigation into our PPO+GPU+Atari+CUDA11.6 tests, @smorad and I have found the problem to be simply the `.backward()` call on the loss. We made sure it's not the GPU->CPU copying, not the optimizer.zero_grad, not the input dtypes (e.g. double instead of float), not the loss math itself, not the model (same number of trainable params than its tf counterpart). We also checked the actual torch graph with the graphviz + torchviz package and saw nothing suspicious. We might have to go back and try different torch+CUDA versions again (we did this once figuring out that CUDA 10.2 was quite good for torch, actually) to overcome these test failures. :/

Reply:
let's hold off on this PR until we come to a conclusion with @sven1977 's investigation on torch + cuda thing.